packages/qwik/src/core/v2/client/vnode-diff.ts

```diff
@@ -562,9 +562,9 @@ export const vnode_diff = (
     }
   }
 
-  /** @param tag Returns true if `qDispatchEvent` needs patching */
-  function createNewElement(jsx: JSXNode, tag: string): boolean {
-    const element = createElementWithNamespace(tag);
+  /** @param elementName Returns true if `qDispatchEvent` needs patching */
```

Comment:
This is misleading.
The return value is only affected by `jsx` argument's `constProps`.

```suggestion
  /** 
   * Returns whether `qDispatchEvent` needs patching. This is true when one of the `jsx` argument's const props has the name of an event.
   * @returns {boolean}
   */
```

Reply:
This is because when we create an element we handle only the constProps. The varProps are handled later

Reply:
Yeah, but the jsdoc is for `function createNewElement(jsx: JSXNode, elementName: string): boolean {`. In which `needsQDispatchEventPatch` is only set to `true` once.

Reply:
yes, I will change the description